plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/attribute-options/editor.scss

```diff
@@ -8,15 +8,15 @@
 }
 
 .wc-block-add-to-cart-with-options-variation-selector-attribute-options__pill {
-	border: 1px solid $black;
-	border-radius: $grid-unit-40;
-	padding: 0.25em 1em;
-	text-transform: uppercase;
+	border: 1px solid currentColor;
+	border-radius: 2px;
+	padding: 0.25em 0.75em;
 	font-size: var(--wp--preset--font-size--small);
```

Comment:
I know this wasn't introduced in this PR, but I just noticed we can remove these styles, as they are already inherited from `style.scss`, which is also loaded in the editor.